apis/fluent/atrium-api-fluent/src/jvmMain/kotlin/ch/tutteli/atrium/api/fluent/en_GB/SQLExceptionFeatures.kt

```diff
@@ -0,0 +1,16 @@
+import ch.tutteli.atrium.api.fluent.en_GB.feature
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.creating.FeatureExpect
+import java.sql.SQLException
+
+val Expect<SQLException>.errorCode: FeatureExpect<SQLException, Int>
```

Comment:
Several things (same same for other val/fun in this file)
- please add KDOC (take a look at collectionFeatureExtractors.kt -> size
  - Add `@since 1.3.0` and link to the sample file
- this way it would be available only for SQLException but not subtypes, change to:
```suggestion
val <T: SQLException> Expect<T>.errorCode: FeatureExpect<T, Int>
```